.github/workflows/py-checkstyle.yml

```diff
@@ -62,7 +62,7 @@ jobs:
 
       - name: Install Ubuntu related dependencies
         run: |
-          sudo apt-get install -y libsasl2-dev unixodbc-dev python3-venv
+          sudo apt-get update  && sudo apt-get install -y libsasl2-dev unixodbc-dev python3-venv
```

Comment:
This is being used in more GH actions. Could you update all of them?

Reply:
Done